migrations/emailmonitor/oxa_email_monitor.py

```diff
@@ -0,0 +1,215 @@
+# This python script determines the failed activation email sends and also creates daily statistics in database
+# This summary data and email failures used in another script for sending status reports and also for sending activation emails again
+
+import MySQLdb
+import collections
+import os
+import re
+from datetime import datetime,timedelta
+
+# Write an error or info log line with timestamp to the application log file /oxa/oxa_mail_monitor.log
+def write_log( log, error=None ):
+    if error == None:
+        os.system("echo '"+str(datetime.now())+" "+ log + "' >> /oxa/oxa_email_monitor.log")
```

Comment:
Let's submit this first PR. Later we can improve it if want to invest more on this script.